tests/functional/firefox/test_accounts.py

```diff
@@ -7,17 +7,17 @@
 from pages.firefox.accounts import FirefoxAccountsPage
 
 
-@pytest.mark.skip_if_firefox(reason='Download button is displayed only to non-Firefox users')
+@pytest.mark.skip_if_firefox(reason='Download button is only displayed to non-Firefox users')
 @pytest.mark.nondestructive
 def test_download_button_displayed(base_url, selenium):
     page = FirefoxAccountsPage(selenium, base_url).open()
-    assert not page.is_accounts_form_displayed
+    assert not page.create_account_button.is_displayed
```

Comment:
I don’t believe `create_account_button` has a property called `is_displayed` as this is something our reusable code to test download buttons has, but not regular elements. I think removing the `.is_displayed` part should be enough for these tests to pass again, but then `assert not page create_account_button` isn’t very readable. I would maybe rename the property to `is_create_account_form_displayed` which sounds a bit clearer.